test/source/tests/page_recipe.ts

```diff
@@ -296,6 +298,17 @@ export class InboxPageRecipe extends PageRecipe {
     await inboxPage.close();
   }
 
+  public static checkFinishingSession = async (t: AvaContext, browser: BrowserHandle, acctEmail: string, threadId: string) => {
+    const inboxPage = await browser.newPage(t, Url.extension(`chrome/settings/inbox/inbox.htm?acctEmail=${acctEmail}&threadId=${threadId}`));
+    await inboxPage.waitAll('iframe');
+    await inboxPage.waitAndClick('@finish-session');
+    let pgpBlockFrame = await inboxPage.getFrame(['pgp_block.htm']);
+    await pgpBlockFrame.waitAll('@pgp-block-content');
+    await pgpBlockFrame.waitForSelTestState('ready');
+    pgpBlockFrame.waitAndClick('@action-show-passphrase-dialog', { delay: 1 });
```

Comment:
Missing await here

Reply:
The missing await could sometimes cause confusion. For example, if the command fails, you could lose the stack trace that failed. Then maybe the tests would fail on some other line later, but you would not know why.

Adding await helps not only wait for completion of the `Promise`, but if there is error, helps immediately throw the error up the stack.

Reply:
Got it, 
Thanks.